1.9/usage/tutorials/dcos-101/loadbalancing.md

```diff
@@ -0,0 +1,37 @@
+---
+post_title: Load-balancing
+nav_title: Load-balancing
+menu_order: 7
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](link to installation instructions].
+* [app2 and Marathon-LB](/docs/1.9/usage/tutorials/dcos-101/app2/) deployed and running in your cluster.
+
+# Objective
+In this final session, we will scale our application to multiple instances and learn how internal and external services choose which instance to use once the application has been scaled.
+
+# Steps
+Load-balancers decide which instance of an app internal or external services should use. With DC/OS, you have two different built-in load-balancer options: : [Marathon-LB](/docs/1.9/usage/service-discovery/marathon-lb/) and [Named VIPs](/docs/1.9/usage/service-discovery/load-balancing-vips/).
+We have seen both before in the context of service discovery and especially marathon-lb when making app2 publically available.
+Now we will explore both options in more detail.
+* First, scale app2 to two instances: `dcos marathon app update /dcos-101/app2 instances=2`
+* Marathon-lb
+    * Check app2 as before via `http://<public-node>10000`. When you do this repeatedly you should see the request being served by different instances of app2.
+    * You can also check the Marathon-LB stats via `http://<public-node>:9090/haproxy?stats`
+* Named VIPs
+    * We have seen named vips before for service discovery inside our cluster
+    * SSH to the leading master node: `dcos node ssh --master-proxy --leader`
+    * `curl dcos-101app2.marathon.l4lb.thisdcos.directory:10000`. When you do this repeatedly you should see the request being served by different instances.
+* Scale app2 back to one instance: `dcos marathon app update /dcos-101/app2 instances=1`
+
+
+
+# Outcome
+We used two different methods to load balance requests for two different instances of our app.
+
+# Deep Dive
+The two load balancing options in DC/OS offer different advantages and drawbacks that you should take into account when choosing one over the other
```

Comment:
`The two load balancing options in DC/OS offer different features and benefits that you should consider when choosing or the other.`